src/main/java/com/laserfiche/repository/api/RepositoryApiClientImpl.java

```diff
@@ -11,7 +12,7 @@
 
 public class RepositoryApiClientImpl implements RepositoryApiClient, AutoCloseable {
     private Map<String, String> defaultHeaders;
-    private UnirestInstance httpClient;
+    private static UnirestInstance httpClient = Unirest.spawnInstance();
```

Comment:
So I think we could keep the old behavior and also keep the part where we initialize the client in the constructor.

Reply:
ok so in this case, i believe i would have to get rid of the both `createFromHttpRequestHandler` functions, since the whole purpose of both those functions is to create an instance and initialize the httpClient

Reply:
@lf-jason-jia ok so now that i removed the `static` keyword, is this comment now resolved?

Reply:
Yeah. I resolved it.